Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -49,6 +50,9 @@
 
 static const wchar_t* TAG = L"CGContext";
 
+// Coordinate offset to support CGGradientDrawingOptions
+static const float s_CGGradientStartPoint = 1E-45;
```

Comment:
this is a "start point" but you are subtracting it from 1 later. That seems like a bad constant name :)